src/workerd/api/pyodide/pyodide.h

```diff
@@ -176,15 +203,22 @@ struct MemorySnapshotResult {
 // CPU architecture-specific artifacts. The logic for loading these is in getArtifacts.
 class ArtifactBundler: public jsg::Object {
 public:
+  kj::Maybe<const PyodidePackageManager&> packageManager;
```

Comment:
Why not create this in ArtifactBundler or pass in a `kj::Own<PyodidePackageManager>`?

Reply:
This is shared between all workers of a WorkerSet (we want it to be shared since it has ownership over the in-memory cache for packages), so owning an instance doesn't make sense here IMO. I believe there's a comment immediately below explaining the ownership and lifetime of this reference.